README.md

```diff
@@ -55,13 +55,14 @@ Optionally set `VROOM_ROUTER=<router>`, `router` being `osrm` (default), `libosr
 
 ## Client side
 
-If `override` is set to `true` in `config.yml`, then the `vroom` command-line parameters `-g`, `-t` and `-x` can be set dynamically per request in order to add detailed route geometry and indicators, set the number of threads and exploration level.
+If `override` is set to `true` in `config.yml`, then the `vroom` command-line parameters `-g`, `-c`, `-t` and `-x` can be set dynamically per request in order to add detailed route geometry and indicators, run vroom in ETA mode, set the number of threads and exploration level.
```

Comment:
In order to be consistent with the docs at https://github.com/VROOM-Project/vroom/blob/master/docs/API.md#solving-mode I think we should use "plan mode" rather than ETA mode in docs and variable names.

Reply:
Okay, I will make changes. 👍